pages/api/v1/user-created-bounties/editBounty.ts

```
@@ -0,0 +1,98 @@
+		const token = getTokenFromReq(req);
+		if (!token) return res.status(401).json({ message: messages?.INVALID_JWT });
+
+		const user = await authServiceInstance.GetUser(token);
+		if (!user) return res.status(401).json({ message: messages.UNAUTHORISED });
+
+		const firebaseFormatedDeadline = new Date(deadlineDate);
+
+		if (!reward || new BN(reward || 0).eq(ZERO_BN)) {
+			return res.status(400).json({ message: 'Invalid reward Amount.' });
+		}
+		if (isNaN(bountyId)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)